cmd/bucket-handlers.go

```diff
@@ -402,12 +409,28 @@ func (api objectAPIHandlers) PostPolicyBucketHandler(w http.ResponseWriter, r *h
 		return
 	}
 
-	fileBody, fileName, formValues, err := extractPostPolicyFormValues(reader)
+	// Read multipart data and save in memory and in the disk if needed
+	form, err := reader.ReadForm(1024 * 1024 * 10)
```

Comment:
> Move this as a constant and btw why 10MB only?

This is an arbitrary value, it means how much memory we allocate per POST multipart request, data that exceeds will be saved in the disk